client/src/pages/automation/project/components/WorkflowExecutionsTestOutput.tsx

```diff
@@ -0,0 +1,155 @@
+                        <ChevronDownIcon className="h-5" />
+                    </div>
+                )}
+            </div>
+
+            <div className="relative size-full">
+                <div className="absolute inset-0 overflow-y-auto">
+                    {workflowIsRunning ? (
+                        <div className="flex size-full items-center justify-center gap-x-1 p-3">
+                            <span className="flex animate-spin text-gray-400">
+                                <RefreshCwIcon className="size-5" />
+                            </span>
+
+                            <span className="text-muted-foreground">Workflow is running...</span>
+                        </div>
+                    ) : workflowTestExecution?.job ? (
+                        <ResizablePanelGroup direction="horizontal">
+                            <ResizablePanel className="overflow-y-auto py-4" defaultSize={30}>
+                                <ul className="divide-y divide-gray-100">
+                                    {triggerExecution && (
+                                        <WorkflowTriggerExecutionItem
+                                            key={triggerExecution.id}
+                                            onClick={() => setContent(triggerExecution)}
+                                            selected={content?.id === triggerExecution.id}
+                                            triggerExecution={triggerExecution}
+                                        />
+                                    )}
+
+                                    {job?.taskExecutions &&
+                                        job?.taskExecutions.map((taskExecution) => (
+                                            <WorkflowTaskExecutionItem
+                                                key={taskExecution.id}
+                                                onClick={() => setContent(taskExecution)}
+                                                selected={content?.id === taskExecution.id}
+                                                taskExecution={taskExecution}
+                                            />
+                                        ))}
+                                </ul>
+                            </ResizablePanel>
+
+                            <ResizableHandle />
+
+                            <ResizablePanel className="space-y-4 overflow-y-auto p-4">
+                                <WorkflowExecutionContent {...content} />
+                            </ResizablePanel>
+                        </ResizablePanelGroup>
+                    ) : (
+                        <div className="flex size-full items-center justify-center gap-x-1 p-3 text-muted-foreground">
+                            <RefreshCwOffIcon className="size-5" />
+
+                            <span>The workflow has not yet been executed.</span>
+                        </div>
+                    )}
```

Comment:
Pošto ode imamo conditional operator na 2 razine za renderanje, ja bi ga razbija u ifove

```tsx
{workflowIsRunning && (...)}

{(!workflowIsRunning && workflowTestExecution?.job) ? (...) : (...)}
```